builder/dimension/dimension.go

```diff
@@ -48,8 +48,20 @@ func Load(data []byte) (builder.Dimension, error) {
 		Typ builder.ComponentType `json:"type,omitempty"`
 	}
 	if err := json.Unmarshal(data, &t); err != nil {
-		return nil, err
+		// the dimension may be a string which is supported by Druid
+		var tstr string
+		if err := json.Unmarshal(data, &tstr); err != nil {
+			return nil, err
+		}
+		// We will just make it a 'default' dimension
+		d = &Base{}
+		d.(*Base).SetType("Default")
+		d.(*Base).SetDimension(tstr)
+		d.(*Base).SetOutputName(tstr)
+		d.(*Base).SetOutputType(types.String)
```

Comment:
The `(*Base)` shouldn't be required in this context, as it's already a concrete type.

Reply:
If I remove the `(*Base)` type assertion, the compiler complains.
<img width="645" alt="image" src="https://user-images.githubusercontent.com/4962911/151899252-b2d61b5e-ceac-43c4-8202-742d6c7406e9.png">

Reply:
/facepalm

My bad, I didn't realize `d` already existed, I read it as `d := &Base{}`.  Keep it as is.